StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -22,11 +22,10 @@ import Foundation
         /// Your implementation should create a PaymentIntent or SetupIntent on your server and call the `intentCreationCallback` with its client secret or an error if one occurred.
         /// - Note: You must create the PaymentIntent or SetupIntent with the same values used as the `IntentConfiguration` e.g. the same amount, currency, etc.
         /// - Parameters:
-        ///   - paymentMethodId: The id of the PaymentMethod representing the customer's payment details.
-        ///     If you need to inspect payment method details, you can fetch the PaymentMethod object using this id on your server. Otherwise, you can ignore this.
+        ///   - paymentMethod: The `STPPaymentMethod` representing the customer's payment details.
```

Comment:
```suggestion
        ///   - paymentMethod: The `STPPaymentMethod` representing the customer's payment details.
        ///   If your server needs the payment method, send `paymentMethod.stripeId` to your server and have it fetch the PaymentMethod object. Otherwise, you can ignore this. Don't send other properties besides the ID to your server.
```